Samples/Hast.Samples.Consumer/Models/ConsumerConfiguration.cs

```diff
@@ -0,0 +1,180 @@
+using Castle.Core.Internal;
+using Hast.Layer;
+using Hast.Samples.Consumer.Attributes;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hast.Samples.Consumer.Models
+{
+    /// <summary>
+    /// This class contains the configuration options you can set through the command line arguments and the terminal

```

Comment:
Ez itt jól van dokumentálva, de a GUI-n nincsenek hintek, és még én se értettem mindent elsőre.

Reply:
![image](https://user-images.githubusercontent.com/4007293/140607232-d53ad124-6220-4928-bff7-20d8da0599a2.png)

Reply:
Fasza! Sample-öknél még javasolnám a kód megtekintését is a usernek.